install.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+waldo_cli_bin="${WALDO_CLI_BIN:-}"
+waldo_cli_url="${WALDO_CLI_URL:-https://github.com/waldoapp/waldo-go-cli/releases/latest/download}"
+
+waldo_exec1_name="waldo"
+waldo_exec2_name="sim_appcenter_build_and_upload.sh"
+
+waldo_asset1_name=
+waldo_asset2_name=
+
+waldo_install_detected=false
+waldo_macos_detected=false
+
+waldo_bash_profile="${HOME}/.bash_profile"
+waldo_bashrc="${HOME}/.bashrc"
+waldo_zshrc="${ZDOTDIR:-${HOME}}/.zshrc"
+
+function check_curl_command() {
+    if ! command -v curl >/dev/null; then
+        fail "No ‘curl’ command found"
+    fi
+}
+
+function check_destination() {
+    local _cli_path=$(command -v waldo)
+    local _cli_bin=
+
+    if [[ -n $_cli_path &&  -e $_cli_path  ]]; then
+        waldo_cli_bin=$(dirname $_cli_path)
+
+        echo "Waldo CLI installation detected in ‘${waldo_cli_bin}’ -- will re-install"
+        echo ""
+
+        waldo_install_detected=true
+    else
+        if [[ -z $waldo_cli_bin ]]; then
+			waldo_cli_bin="${HOME}/.waldo/bin"
+        fi
+
+        echo "Waldo CLI will be installed in ‘${waldo_cli_bin}’"
+        echo ""
+    fi
+}
+
+function check_installation() {
+    if [[ $waldo_install_detected == true ]]; then
+        echo ""
+        echo "Waldo CLI successfully re-installed!"
+    else
+		if [[ $waldo_macos_detected == true ]]; then
+			touch "$waldo_bash_profile"
```

Comment:
Not sure i'm a fan of us creating theses files if bash isn't used. They don't exist on recent macOS versions and we force their creation for nothing.

(Other installers also seem to do it as the file was force-created on my system)